### ISSUE TITLE
YT not connecting

### ISSUE THREAD

**Issue by Hurzchen** [steveseguin/social_stream#147]
Twitch and Kick connect without any problem, but YT does not connect. Not before and not after starting a stream and no matter what username i enter. What username should be entered and what format (with the @ or without)? Also YT does not work with the chrome plugin either.

PS: i love the idea behind this software and with not Twtich dropping the exclusivity the need for it will rise.

**Comment by Littlepush35** [comment]
Hello, I have the same problem. Everything works fine except for Youtube, it tells me that the live is not active even though it is running.

**Comment by steveseguin** [comment]
Thank you for reporting the issues. I'll be looking into it.

**Comment by steveseguin** [comment]
You can create an "Other chat sites" source, and put into your chat pop out URL into that, as a workaround for YT for now

**Comment by Hurzchen** [comment]
I had the same issue with restream chatbot (i am looking for a relay chat bot that writes Twitch into YT and YT into Twitch Chat) and found out, that YT live streams are not recognized when added as live streams, but when added as events. Then the Bot does get the "stream is live" info and the chat.

I guess YT did change a thing or two here that prevents the correct recognition of a stream on the channel being live.

**Comment by onsterion** [comment]
Yep, same problem, tried my youtube username in all kind of ways and not luck :'(

Twitch, kick and tiktok works fine.

**Comment by Littlepush35** [comment]
> You can create an "Other chat sites" source, and put into your chat pop out URL into that, as a workaround for YT for now

This is what I do, but since the address changes each time, it requires recreating a new source when launching live. But it's a good solution in the meantime.

**Comment by Lars4k** [comment]
Same here :(
